src/util/loanUtils.js

```diff
@@ -93,21 +93,60 @@ export function build5DollarsPriceArray(amountLeft) {
 	return priceArray;
 }
 
-export function getDropdownPriceArray(unreservedAmount, minAmount, enableFiveDollarsNotes, inPfp = false) {
+function buildHugePriceArray(amountLeft) {
```

Comment:
Since these utils are critical in our app, new unit tests should be added for these changes.

Reply:
Agreed! I'm on it.

Reply:
Added!